contrib/win32/win32compat/fileio.c

```diff
@@ -738,12 +738,21 @@ fileio_stat(const char *path, struct _stat64 *buf)
 	wchar_t* wpath = NULL;
 	WIN32_FILE_ATTRIBUTE_DATA attributes = { 0 };
 	int ret = -1, len = 0;	
+
+	memset(buf, 0, sizeof(struct _stat64));
+
+	/* Detect root dir */
+	if (path && strcmp(path, "/") == 0) {
```

Comment:
from winscp (or) any other third party software.. if we receive "/" then here it is treated as the root dir....

![image](https://cloud.githubusercontent.com/assets/23668037/26429965/17a3478a-409f-11e7-8c25-d771455f20f3.png)